src/wasm/wasm-binary.cpp

```diff
@@ -1847,6 +1847,7 @@ void WasmBinaryBuilder::readTypes() {
   size_t numTypes = getU32LEB();
   BYN_TRACE("num: " << numTypes << std::endl);
   TypeBuilder builder(numTypes);
+  builder.createRecGroup(0, numTypes);
```

Comment:
What is this?

Reply:
Oh, oops, this is one my local hacks to get measurements. Will remove.